flash/tabular/forecasting/data.py

```diff
@@ -151,13 +144,13 @@ def from_data_frame(
         val_transform: Optional[Dict[str, Callable]] = None,
         test_transform: Optional[Dict[str, Callable]] = None,
         predict_transform: Optional[Dict[str, Callable]] = None,
-        data_fetcher: Optional[BaseDataFetcher] = None,
-        input_transform: Optional[InputTransform] = None,
+        # TODO: Update these when DataModule is updated
         val_split: Optional[float] = None,
         batch_size: int = 4,
-        num_workers: Optional[int] = None,
-        **input_transform_kwargs: Any,
-    ):
+        num_workers: int = 0,
+        sampler: Optional[Type[Sampler]] = None,
+        **time_series_dataset_kwargs: Any,
```

Comment:
This should be datamodule kwargs no ?

Reply:
Unfortunately I think it has to be like this. The `TimeSeriesDataset` object from pytorch forecasting that these kwargs are used for as like a hundred different options so it's not really possible to just list them all here. What we could maybe do is have the time series kwargs as a dict instead, but I'd like to leave that for a future PR as it would require us to also update our demos.